typeguard/__init__.py

```diff
@@ -436,7 +454,8 @@ def check_class(argname: str, value, expected_type, memo: _TypeCheckMemo) -> Non
     if expected_type is Type:
         return
 
-    expected_class = expected_type.__args__[0] if expected_type.__args__ else None
+    expected_class = expected_type.__args__[0] if hasattr(
```

Comment:
If a tertiary expression has to be split to multiple lines then it should be converted to a real "if" block.